### ISSUE TITLE
Multiple Locations

### ISSUE THREAD

**Issue by fuzzysb** [home-assistant/home-assistant.github.io#26391]
### Feedback

I am failing to see why multiple locations cannot be catered for, you have the index value, ideally the parameter should be indexes to exclude, by default all locations are brought into home assistant

### URL

https://www.home-assistant.io/integrations/evohome/

### Version

2023.2.5

**Comment by zxdavb** [comment]
Thank you for your submission. 

TLDR: https://github.com/zxdavb/evohome-async/wiki/Hass:-Multiple-Evohome-Locations

FYI: there is an existing workaround for this feature discussed in the [Community Forum](https://community.home-assistant.io/t/workaround-evohome-multiple-concurrent-locations/161353/17)

This is an issue with **evohome** and so should have been submitted at (https://github.com/home-assistant/core/issues). However, this is not a bug - it is a feature request. 

The HA team use GitHub for tracking issues, not for providing support or tracking feature requests.

So, I will close this request.

If you want to suggest a feature, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

Or you can submit a PR for review.

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍
